unittests/test_functions.py

```diff
@@ -17,18 +17,11 @@
     TestRunnerException
 )
 from compliance_suite.functions.client import Client
-from compliance_suite.functions.log import set_logging
 from compliance_suite.functions.report import Report
 
 
 class TestFunctions(unittest.TestCase):
 
-    def test_set_logging(self):
-        """ Checks if the logger is set up properly """
-
-        set_logging()
-        assert True
-
```

Comment:
There's a whole new class with a number of methods. I know they are rather trivial, but would be good to provide tests for them anyway.

I know that coverage doesn't decrease. But this is as a side effect of testing some _other_ units of code. Ideally we should have dedicated unit tests for each unit of code.